librz/core/cmd.c

```diff
@@ -4780,20 +4780,86 @@ DEFINE_HANDLE_TS_FCN_AND_SYMBOL(redirect_command) {
 	return res;
 }
 
+static bool do_help_manual(RzCmd *cmd, const char* cmdname) {
+	char *cmdname_help = NULL;
+	char *help_msg = NULL;
+	RzCmdParsedArgs *help_pra = NULL;
+	if (cmdname) {
```

Comment:
can `cmdname` really be NULL? If it can (i'm not sure), i'd prefer a `if (!cmdname)` and an early return rather than having everything indented.